### PR TITLE
[cilium-hubble] migrate cilium-hubble images to distroless

### PR DESCRIPTION
**Pull request by AndreyPavlovFlant**
## Description

`cilium-hubble` is based on a distroless images.

## Why do we need it, and what problem does it solve?

We'd like to base our images on a distroless image.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cilium-hubble
type: chore
summary: cilium-hubble is now based on a distroless images.
impact: cilium-hubble pods will restart.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/static'...

File: modules/500-cilium-hubble/images/ui-frontend/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-base-artifact
+from: quay.io/cilium/hubble-ui:v0.12.1@sha256:e15af59a2ded739e420be82de6cbdd0ce22f8d3f00e3a10b3d2d2734e074a394
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}
+fromImage: common/nginx-static
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-base-artifact
+  add: /etc/nginx/conf.d/default.conf
+  to: /opt/nginx-static/conf/nginx.conf
+  before: install
```
Comment (apolovov) [inline modules/500-cilium-hubble/images/ui-frontend/werf.inc.yaml:11]: get rid of this file pls

**Review by apolovov — CHANGES_REQUESTED** [review]
A single fix

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/static'...

**Comment by AndreyPavlovFlant** [comment]
/e2e/abort static;Static;containerd;1.25 7581139851 7581139851-1-con-1-25 /sys/deckhouse-oss/install:pr7174 astra@192.168.199.109

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/static'...

**Comment by AndreyPavlovFlant** [comment]
/e2e/abort static;Static;containerd;1.25 7581607776 7581607776-1-con-1-25 /sys/deckhouse-oss/install:pr7174 astra@192.168.199.162

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by AndreyPavlovFlant** [comment]
/e2e/abort static;Static;containerd;1.25 7583203576 7583203576-1-con-1-25 /sys/deckhouse-oss/install:pr7174 astra@192.168.199.97

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/openstack'...

**Comment by AndreyPavlovFlant** [comment]
/e2e/abort openstack;Standard;containerd;1.25 7584533314 7584533314-1-con-1-25 /sys/deckhouse-oss/install:pr7174 debian@45.12.231.84

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/openstack'...

**Review by apolovov — APPROVED** [review]

**Pull request merged**